docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -185,22 +252,25 @@ The remapping values for each globally cached lookup can be specified by a json
 |`uriPrefix`|A URI which specifies a directory (or other searchable resource) in which to search for files|No|Use `uri`|
 |`fileRegex`|Optional regex for matching the file name under `uriPrefix`. Only used if `uriPrefix` is used|No|`".*"`|
 |`namespaceParseSpec`|How to interpret the data at the URI|Yes||
+|`keyValueMaps`|Fields used in lookup maps inside the lookup|No|`{"mapName":"default","keyName":"key","valueName":"value"}`|
 
 One of either `uri` xor `uriPrefix` must be specified.
 
 The `pollPeriod` value specifies the period in ISO 8601 format between checks for replacement data for the lookup. If the source of the lookup is capable of providing a timestamp, the lookup will only be updated if it has changed since the prior tick of `pollPeriod`. A value of 0, an absent parameter, or `null` all mean populate once and do not attempt to look for new data later. Whenever an poll occurs, the updating system will look for a file with the most recent timestamp and assume that one with the most recent data set, replacing the local cache of the lookup data.
 
 The `namespaceParseSpec` can be one of a number of values. Each of the examples below would rename foo to bar, baz to bat, and buck to truck. All parseSpec types assumes each input is delimited by a new line. See below for the types of parseSpec supported.
 
+The `keyValueMaps` specifies lookup maps under one lookup name. One lookup can have multiple maps that share the source of lookup. It is a list of maps called `keyValueMap` that has three entries, `mapName`, `keyName`, and `valueName`. `mapName` is the name of map inside the lookup and `keyName` is key column or field name and `valueName` is value column or field name.
```

Comment:
Suggest adding a bit more documentation detail along the lines of:

- Key/Value column refer to columns within the lookup source; "columns" field refers to Druid columns whose values will be used as filtering criteria for retrieving the mapping row from the lookup source